kubeflow/core/tf-job.libsonnet

```diff
@@ -280,8 +280,8 @@
               "apiVersion: ambassador/v0",
               "kind:  Mapping",
               "name: tfjobs-ui-mapping",
-              "prefix: /tfjobs/ui/",
-              "rewrite: /",
+              "prefix: /tfjobs/",
+              "rewrite: /tfjobs/",
```

Comment:
Can you add a comment explaining  why prefix and rewrite are set the way they are? I would like to understand better what it takes to make Ambassador work. Does the TFJobs UI expect that is rooted at "/"?

Reply:
Please see explanation here: https://github.com/kubeflow/tf-operator/pull/441#discussion_r173168960

Reply:
@jlewi Did you have a chance to look at my explanations?